### PR TITLE
Add 5 fields to the schema

### PR DESCRIPTION
**Pull request by ambal007**

### CODE REVIEW

**Comment by tanderegg** [comment]
@mattboehm This looks about right to me, could you take a look as well?  Thanks!

File: ckanext/cfpb_extrafields/plugin.py [hunk 1]
```diff
@@ -275,6 +273,16 @@ def _modify_package_schema(self, schema):
                                       tk.get_converter('convert_to_extras'),],
             'privacy_has_direct_identifiers' : [tk.get_validator('ignore_missing'),
                                                 tk.get_converter('convert_to_extras'),],
+            'transfer_date' : [tk.get_validator('ignore_missing'),
+                                                tk.get_converter('convert_to_extras'),],
+            'data_governance_notes' : [tk.get_validator('ignore_missing'),
+                                                tk.get_converter('convert_to_extras'),],
+            'legal_notes' : [tk.get_validator('ignore_missing'),
+                                                tk.get_converter('convert_to_extras'),],
+            'pra:_notes' : [tk.get_validator('ignore_missing'),
```
Comment (mattboehm) [inline ckanext/cfpb_extrafields/plugin.py:282]: We need to remove the extra colon from this line

File: ckanext/cfpb_extrafields/plugin.py [hunk 2]
```diff
@@ -275,6 +273,16 @@ def _modify_package_schema(self, schema):
...
+                                                tk.get_converter('convert_to_extras'),],
```
Comment (mattboehm) [inline ckanext/cfpb_extrafields/plugin.py:283]: Need to double check whether each of these 5 fields should have these 2 converters

File: ckanext/cfpb_extrafields/plugin.py [hunk 3]
```diff
@@ -395,7 +401,16 @@ def show_package_schema(self):
...
             'cleansing_rules_used' : [tk.get_converter('convert_from_extras'),
-                                      tk.get_validator('ignore_missing'),],
+                                                 tk.get_validator('ignore_missing'),],
+            'transfer_date' : [tk.get_validator('ignore_missing'),
+                                                tk.get_converter('convert_from_extras'),],
+            'data_governance_notes' : [tk.get_validator('ignore_missing'),
+                                                tk.get_converter('convert_from_extras'),],
+            'legal_notes' : [tk.get_validator('ignore_missing'),
+                                                tk.get_converter('convert_from_extras'),],
+            'pra:_notes' : [tk.get_validator('ignore_missing'),
```
Comment (mattboehm) [inline ckanext/cfpb_extrafields/plugin.py:411]: extra colon

File: ckanext/cfpb_extrafields/templates/package/snippets/additional_info.html [hunk 4]
```diff
@@ -281,10 +281,34 @@
           <td class="dataset-details">{{ pkg_dict.privacy_has_direct_identifiers }}</td>
         </tr>
         {% endif %}
-        {% if pkg_dict.privacy_has_privacy_act_statement %}
+        {% if transfer_date %}
```
Comment (mattboehm) [inline ckanext/cfpb_extrafields/templates/package/snippets/additional_info.html:284]: These probably need to be `pkgdict.transfer_date`

File: ckanext/cfpb_extrafields/templates/package/snippets/package_metadata_fields.html [hunk 5]
```diff
@@ -161,9 +161,21 @@
 label=_('privacy: has direct identifiers?'),
 current_value=data.privacy_has_direct_identifiers)}}
 
-{{ form_custom.radio('privacy_has_privacy_act_statement',
-label=_('privacy: has Privacy Act statement'),
-current_value=data.privacy_has_privacy_act_statement)}}
+{{ form_custom.radio('transfer_date',
```
Comment (mattboehm) [inline ckanext/cfpb_extrafields/templates/package/snippets/package_metadata_fields.html:164]: These should not be radio inputs

File: ckanext/cfpb_extrafields/templates/package/snippets/package_metadata_fields.html [hunk 6]
```diff
@@ -215,27 +227,32 @@
...
+      "privacy_notes": "Any additional information related to the privacy metadata",
       "privacy_sorn_number": "If applicable, the title of the applicable System of Record Notice (SORN). For a full list of the Bureau's SORN's, go to http://www.consumerfinance.gov/privacy/",
       "procurement_document_id": "If applicable, the unique identifier from the Office of Procurement's internal database",
       "records_retention_schedule": "The relevant retention schedule or other information about retention requirements",
       "relevant_governing_documents": "Which types of documents govern this dataset",
       "sensitivity_level": "Bureau-assigned sensitivity level per the Information Sensitivity Leveling Standard. For more information, see http://team.cfpb.local/wiki/index.php/Information_Sensitivity_Leveling_Standard",
       "tags": "High-level subject matter that the dataset is related to. This creates tags that help group related datasets.",
+      "transfer_date": "Initial date where the data was brought into the Bureau",
```
Comment (mattboehm) [inline ckanext/cfpb_extrafields/templates/package/snippets/package_metadata_fields.html:255]: s/where/when

**Review by mattboehm — CHANGES_REQUESTED** [review]
I'll need to make a handful of changes to this and then try testing it out on staging.

**Pull request closed without merging**